converter.py

```diff
@@ -0,0 +1,183 @@
+NSTAT       {nstat:d}    -1 ! NSTAT, Step of saving
+STRAGG          2            ! Straggling: 0-Off 1-Gauss, 2-Vavilov
+MSCAT           2            ! Mult. scatt 0-Off 1-Gauss, 2-Moliere
+NUCRE           0            ! Nucl.Reac. switcher: 1-ON, 0-OFF
+"""
+
+    def __init__(self, energy: float = 250., nstat: int = 10000) -> None:
+        self.energy = energy
+        self.nstat = nstat
+
+    def __str__(self) -> str:
+        return self.beam_template.format(energy=self.energy, nstat=self.nstat)
+
+
+class MatConfig:
+    """Class mapping of the mat.dat config file"""
+
+    material_template: str = """MEDIUM {idx}
+ICRU {mat}
+END
+"""
+
+    def __init__(self, materials: list[int] = [276]) -> None:
+        self.materials: list[str] = materials
+
+    def __str__(self) -> str:
+        material_strings = [self.material_template.format(
+            idx=idx, mat=mat) for idx, mat in enumerate(self.materials)]
+        return "\n".join(material_strings)
+
+
+class DetectConfig:
+    """Class mapping of the detect.dat config file"""
+    detect_template = """Output
+    Filename mesh.bdo
+    Geo ScoringCylinder
+    Quantity Dose
+    """
+
+    def __init__(self, geometries: list[Geometry] = [Cylinder(), Mesh()]) -> None:
+        self.geometries: Geometry = geometries
+
+    def __str__(self):
+        detect_strings = [str(geo)
+                          for geo in self.geometries] + [self.detect_template]
+        return "\n".join(detect_strings)
+
+
+class GeoConfig:
+    # TODO: Parametrize this
+    """Class mapping of the geo.dat config file"""
+
+    geo_template: str = """
+*---><---><--------><------------------------------------------------>
+    0    0           protons, H2O 30 cm cylinder, r=10, 1 zone
+*---><---><--------><--------><--------><--------><--------><-------->
+  RCC    1       0.0       0.0       0.0       0.0       0.0      30.0
+                10.0
+  RCC    2       0.0       0.0      -5.0       0.0       0.0      35.0
+                15.0
+  RCC    3       0.0       0.0     -10.0       0.0       0.0      40.0
+                20.0
+  END
+  001          +1
+  002          +2     -1
+  003          +3     -2
+  END
+* material codes: 1 - liquid water (ICRU material no 276), 1000 - vacuum, 0 - black body
+    1    2    3
+    1 1000    0
+"""
+
+    def __init__(self) -> None:
+        pass
+
+    def __str__(self) -> str:
+        return self.geo_template
+
+
+class Parser(ABC):
+    @abstractmethod
+    def parse_configs(json: dict):
+        pass
+
+    @abstractmethod
+    def print_configs(target_dir_path: str):
+        pass
+
+
+class DummmyParser(Parser):
+    def __init__(self) -> None:
+        self.beam_config = BeamConfig()
+        self.mat_config = MatConfig()
+        self.detect_config = DetectConfig()
+        self.geo_config = GeoConfig()
+
+    def parse_configs(self, json: dict):
+        pass
+
+    def print_configs(self, target_dir: str):
```

Comment:
```suggestion
    def save_configs(self, target_dir: str):
```

Reply:
Resolved in 3745118fadacd3eefeaf2cee5b591052a549786f